stock_scanner/models/scanner_hardware.py

```diff
@@ -311,7 +305,6 @@ def _scanner_call(self, action, message=False,
         # Nothing matched, return an error
         return self._send_error(['Unknown', 'action'])
 
-    @api.multi
```

Comment:
Any reason for removing the `api.multi`s? Explicit is better IMO

Reply:
I always remove `@api.multi` when migrating to v10.0 or higher because this is now the default.
Is there a guideline about this ?